src/testFixtures/workspaceFolder/java11-image-gradle-sam-app/README.md

```diff
@@ -0,0 +1,115 @@
+# java11-image-gradle-sam-app
```

Comment:
Do we need the readme for testdata? or the test/event for that matter?

Reply:
Can probably strip it out, will wait until I know integration tests are working before cleanup.